app/controllers/further_education_payments/providers/claims/information_controller.rb

```diff
@@ -3,7 +3,7 @@ module Providers
     module Claims
       class InformationController < BaseController
         def show
-          @claim = claim_scope.find(params[:claim_id])
+          @claim = current_provider.claims.find(params[:claim_id])
```

Comment:
we do lose the academic year I believe here so in future may show more claims than desired?